.vscode/settings.json

```diff
@@ -22,5 +22,6 @@
 
     // Because we use Flow - see https://github.com/flowtype/flow-for-vscode#setup
     "javascript.validate.enable": false,
-    "clang.compilationDatabase": "${workspaceRoot}/Example/compile_commands.json"
+    "clang.compilationDatabase": "${workspaceRoot}/Example/compile_commands.json",
+    "flow.pathToFlow": "./node_modules/.bin/flow"
```

Comment:
Flow is now hosted inside npm, instead of relying on your global scope, given that everyone could have any random version installed from brew, this is 👍